LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
General curiosity question: Is there a specific reason we use the Apache License by default, instead of, say MIT – or does it just come down to personal preference?